compute_sdk/globus_compute_sdk/sdk/client.py

```diff
@@ -752,16 +752,22 @@ def get_endpoint_metadata(self, endpoint_uuid):
             return self._compute_web_client.v2.get_endpoint(endpoint_uuid).data
 
     @_client_gares_handler
-    def get_endpoints(self):
+    def get_endpoints(self, role: str | None = None):
         """Get a list of all endpoints owned by the current user across all systems.
 
+        Parameters
+        ----------
+        role: str
+            Role of the user in relation to the endpoint queried. (e.g.: 'owner', 'any')
+            By default only endpoints that are owned by the user are fetched.
```

Comment:
By default, this looks like the SDK does not specify a default, and lets the API return a default.  Suggest being more precise about that description.  Perhaps something like:

> The default value is unset, in which case the API (currently) assumes ``owner``.

Could potentially link to the API route documentation as well.

Reply:
Fixed.